libs/ui-lib/lib/ocm/components/clusterConfiguration/operators/MceCheckbox.tsx

```diff
@@ -7,24 +7,43 @@ import { useNewFeatureSupportLevel } from '../../../../common/components/newFeat
 
 const MCE_FIELD_NAME = 'useMultiClusterEngine';
 
-const MceLabel = ({ disabledReason }: { disabledReason?: string }) => (
-  <>
-    <Tooltip hidden={!disabledReason} content={disabledReason}>
-      <span>Install multicluster engine </span>
-    </Tooltip>
-    <PopoverIcon
-      id={MCE_FIELD_NAME}
-      component={'a'}
-      headerContent="Additional requirements"
-      bodyContent={
-        <>
-          OpenShift Data Foundation (recommended for creating additional on-premise clusters) or
-          another persistent storage service
-        </>
-      }
-    />
-  </>
-);
+const MceLabel = ({
+  disabledReason,
+  showMessageForLvms,
+  isSNO,
+}: {
+  disabledReason?: string;
+  showMessageForLvms?: boolean;
+  isSNO?: boolean;
+}) => {
+  const odfText = !isSNO
+    ? 'OpenShift Data Foundation (recommended for creating additional on-premise clusters)'
+    : '';
+  const lvmsText = showMessageForLvms
+    ? !isSNO
+      ? ', Logical Volume Manager Storage'
+      : 'Logical Volume Manager Storage'
+    : '';
```

Comment:
Same problems as in `CnvHostRequirements`. Is it the same message using the same conditions by the way?

If so, why not extract it to an external function to `utils.tsx`?